<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDlg</class>
 <widget class="QDialog" name="AdminDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>398</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date_query"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>时间类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cobox_queryType">
          <item>
           <property name="text">
            <string>按日查询</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按月查询</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>优惠选项</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cobox_choose">
          <item>
           <property name="text">
            <string>所有</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>成人</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>小孩</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>老人</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>团体</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="edit_reslut"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_query">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_clear">
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
